### PR TITLE
Updating the Math and MathF docs to indicate which functions may vary across OS or architecture

### PR DESCRIPTION
**Pull request by tannergooding**
This resolves https://github.com/dotnet/dotnet-api-docs/issues/4231

### CODE REVIEW

**Comment by tannergooding** [comment]
How do you requeue the `OpenPublishing.Build` job? It looks like an unrelated failure.

**Comment by gewarren** [comment]
@tannergooding You close/reopen the PR. I'll do that now.

File: xml/System/Math.xml [hunk 1]
```diff
@@ -616,6 +616,8 @@
 ## Remarks  
  Multiply the return value by 180/<xref:System.Math.PI?displayProperty=nameWithType> to convert from radians to degrees.  
   
+ This method calls into the underlying C runtime and the exact result or valid input range may differ between different Operating Systems or Architectures.
```
Comment (gewarren) [inline xml/System/Math.xml:619]:
```suggestion
 This method calls into the underlying C runtime, and the exact result or valid input range may differ between different operating systems or architectures.  
```

Comment (tannergooding) [reply xml/System/Math.xml]: Applied to all occurrences of the sentence.

**Review by gewarren — APPROVED** [review]
Looks good, thanks.

**Pull request merged**